acme/setup.py

```diff
@@ -14,7 +14,7 @@
     'PyOpenSSL>=17.3.0',
     'pyrfc3339',
     'pytz',
-    'requests>=2.6.0',
+    'requests>=2.14.2',
```

Comment:
Seeing these makes me think that maybe the pinning document could use a reminder that minimum required versions are still specified in `setup.py` files.